### DISCUSSION TITLE
V3 Pro & Pan Pro

### DISCUSSION THREAD

**Discussion by dmkjr** [mrlt8/docker-wyze-bridge#546] [General]
I did a quick Google search and don't see these being released or publicly announced..... do you have anymore information on them or where they just found in the API?

**Comment by mrlt8** [comment]
They're from the release version of the wyze app and are the only two cams that seem have "isSupport2KVideo". 

There are a couple of other unreleased cams and networking/iot devices in the app.

**Comment by dmkjr** [comment]
I'm interested in their new products, I just hope the software gets better and more integrated.  We all give Apple shit, but at least their products work seamlessly.  The vacuum's were a disaster (from a software perspective).  Speaking of that, I'm using another integration for Wyze Vacuum into HA.  Does your Wyze Bridge do that as well?

**Reply by mrlt8** [reply]
I don't think so. The bridge is primarily based on Throughtek's p2p sdk that a lot of other manufactures also use to connect and stream from the cameras to the app. 

I haven't really looked into the vacuums, but from what I can tell, the vacuums are using their "Venus" platform. Same with the Doorbell Pro which seems to be made by gwell and is using their "mars" platform which seems to be webrtc based. 

I think they're trying to get all their devices to revolve around the two wifi 6/e mesh routers aka the "sun".

**Comment by dmkjr** [comment]
Interested in whatever other products they are creating.  While I like the
cams, I wish they had more “thought” into their software and how things can
integrate and work with each other.
